src/jumio-kyc/utils/serialize-kyc.ts

```diff
@@ -12,12 +13,17 @@ export function serializeKyc(
   canAttemptReason: string,
   canCreate: boolean,
   canCreateReason: string,
+  config: ApiConfigService,
 ): SerializedKyc {
   assert.ok(redemption.jumio_account_id);
 
+  const maxAttempts =
```

Comment:
Feels a little weird to pass an injected dependency into a serializer. I think it'd be cleaner if we did this in the caller (or just passing the config value as a default)

Reply:
That is a better way but it it requires every test to change that tests this to do that